experiments/interpretation/dutchf3_patch/local/azureml_requirements.txt

```diff
@@ -0,0 +1,13 @@
+git+https://github.com/olgaliak/seismic-deeplearning.git@azureml-train-pipeline#egg=cv_lib&subdirectory=cv_lib
```

Comment:
![image](https://user-images.githubusercontent.com/8856246/74265133-d2a4dc80-4cb6-11ea-8e81-64b6fa02717a.png)
Anna added an empty __init__.py to cv_lib/cv_lib/segmentation/dutchf3 to be able to import the scripts in dutchf3. So I believe that is why she has the path here pointing to this branch instead of master. Although I agree this modification should take place once it is merged. I am thinking after this PR is complete then we can have a task to submit a follow up PR to update this path?

Reply:
@maxkazmsft

Reply:
@annazietlow

Reply:
Yep, that's the reason and I think your approach with a separate PR makes a lot of sense. Thanks, Kira!

Reply:
Thanks and sounds good, taking care of this in #202

Reply:
This has been updated to only pull from /master. See below, closing this conversation. 
![image](https://user-images.githubusercontent.com/8856246/74392011-aa9ea180-4dba-11ea-8b26-638f5800b0ba.png)